terraform/deployments/apps/common.tf

```diff
@@ -10,6 +10,7 @@ locals {
   static_url                       = "https://static.${var.mesh_domain}"
   statsd_host                      = "statsd.${var.mesh_domain}" # TODO: Put Statsd in App Mesh
   website_root                     = "https://www.${var.app_domain}"
+  router_urls                      = "router.${var.mesh_domain}:3055"
```

Comment:
This can't just be a load-balanced address for reaching an arbitrary router instance; it has to list all of the instances individually. Otherwise the route tree will be nearly always inconsistent because Router API will only be able to send a reload request to one arbitrarily-chosen Router instance each time there's a change, instead of reloading them all.